ivadomed/training.py

```diff
@@ -430,8 +430,7 @@ def get_loss_function(params):
                                "BinaryCrossEntropyLoss", "TverskyLoss", "FocalTverskyLoss", "AdapWingLoss", "L2loss",
                                "LossCombination"]
     if loss_name not in loss_function_available:
-        print("Unknown Loss function, please choose between {}".format(loss_function_available))
-        exit()
+        raise ValueError("Unknown Loss function, please choose between {}".format(loss_function_available))
```

Comment:
I would recommend printing the unknown loss function name

Reply:
done in 
a3e884d